README.md

```diff
@@ -51,6 +51,14 @@ Open the `/modules/.../<file_name>.ino` file in Arduino, select the appropriate
 
 The stm32 modules have debug harnessing built in; for instance, running `cmake --build ./build-stm32-cross --target heater-shaker-debug` will build the firmware and spin up the cross gdb and openocd. This target by default (because of commands in the gdbinit) will reset the board and upload the built firmware; if you want to use the firmware currently flashed to the board, run everything manually.
 
+## test
+
+This is not implemented for the arduino modules.
+
+For the STM32 modules, if you configure the host toolchain: `cmake --preset=stm32-host .`, you can then run the `check` target: `cmake --build ./build-stm32-host --target build-and-test`, which will build the tests and then run them. You can also use cmake's built-in `test` target: `cmake --build ./build-stm32-host --target test`, but this will not automatically actually build the tests.
```

Comment:
Should this be the `build-and-test` target now that it has been renamed?
```suggestion
For the STM32 modules, if you configure the host toolchain: `cmake --preset=stm32-host .`, you can then run the `build-and-test` target: `cmake --build ./build-stm32-host --target build-and-test`, which will build the tests and then run them. You can also use cmake's built-in `test` target: `cmake --build ./build-stm32-host --target test`, but this will not automatically actually build the tests.
```